website/docs/language/import/index.mdx

```diff
@@ -0,0 +1,188 @@
+---
```

Comment:
What file structure are you intending here? Do you want to have two pages, like so:
```
Import
-> Overview
-> Generating Configuration
```

If yes, can you update the `website/data/language-nav-data.json` to the following:
```
{
      "title": "Import",
      "routes": [
          { "title": "Overview", "path": "import/index" },
          {
            "title": "Generating Configuration",
            "path": "import/generating-configuration"
          }
        ]
  },
```

Reply:
(Resolving this in favor of a discussion on the `language-nav-data.json` file itself.)